static/templates/user_sidebar_actions.handlebars

```diff
@@ -18,5 +18,11 @@
       {{#tr this}}Compose a message to <b>__name__</b>{{/tr}}
     </a>
   </li>
+  <li>
+    <a class="star_user">
+      <i class="icon-vector-star"></i>
+      {{#tr this}}{{#if starred}}{{t Unstar}}{{else}}{{t Star}}{{/if}} {{t user}} <b>__name__</b>{{/tr}}
```

Comment:
I think the point is that both `{{#tr this}}` and `{{t Star}}` are ways of tagging for i18n, and we only need one of those....